clang/test/SemaCXX/coroutines.cpp

```diff
@@ -3,6 +3,7 @@
 
 // RUN: %clang_cc1 -std=c++23 -fsyntax-only -verify=expected,cxx20_23,cxx23    %s -fcxx-exceptions -fexceptions -Wunused-result
 // RUN: %clang_cc1 -std=c++20 -fsyntax-only -verify=expected,cxx14_20,cxx20_23 %s -fcxx-exceptions -fexceptions -Wunused-result
+// RUN: not %clang_cc1 -std=c++20 -fsyntax-only %s -fcxx-exceptions -fexceptions -Wunused-result 2>&1 | FileCheck %s
```

Comment:
Could you add a comment here explaining to the reader why we need this? It's unusual to have `-verify` on some `RUN` lines and not have it on others.

Something like

```
// Run without -verify to check the order of errors we show.
// RUN: not ...
```

Reply:
Done.